lib/msf/core/handler/reverse_http.rb

```diff
@@ -355,6 +357,20 @@ def bind_port
     port > 0 ? port : datastore['LPORT'].to_i
   end
 
+  def get_ssl_cert_hash
```

Comment:
The reverse_http handler is abused as a base class for most of the HTTP-based payloads (including SSL), so this makes sense.